### DISCUSSION TITLE
Why can't we have named slots for layouts?

### DISCUSSION THREAD

**Discussion by novotionyx** [sveltejs/kit#6873] [Ideas]
I've found this feature to be non existent yet constantly I run into it. Layouts are inherintly just components that take on children, and in any other context (such as writing our own component) - if we have children, we can allow named slots and place things in specific positions using `<svelte:fragment>` or by simply targeting the named slots with any other tag.

I think if we had named slots for layouts, it would reduce the tendency to create custom Layout components to compensate. I'm sure this idea has been brought up before, I just find I run into this shortcoming a lot more than I'd like!

**Comment by AdaptingAFM** [comment]
[This feature](https://github.com/sveltejs/kit/issues/627) has already been considered and the core maintainers gave it a to-implement priority of `whenever` (see: https://github.com/sveltejs/kit/milestone/3); if you want this to be implemented as soon as possible, raise concern or implement it on your own and push a pull request.
